README.md

```diff
@@ -165,7 +164,11 @@ class future {
 };
 
 template <typename S, typename F, typename... Ts>
-auto when_all(S, F, future<Ts>... args); // -> future<result_of_t<F(Ts...>>
+auto when_all(S, F, future<Ts>... args); // -> future<result_of_t<F(Ts...)>>
+
```

Comment:
How do the signature of these two cases differ? I think maybe the second one is when_any?